integration_tests/test_rough_align.py

```diff
@@ -534,192 +464,40 @@ def test_apply_rough_alignment_transform(render, montage_stack, test_do_rough_al
-        assert(1021 not in zvalues)
-
-
-def make_montage_stack_without_downsamples(render, montage_stack, tmpdir_factory):
-    # testing for make montage scape stack without having downsamples generated
-    tmp_dir = str(tmpdir_factory.mktemp('downsample'))
-    output_stack = '{}_Downsample'.format(one_tile_montage)
-    params = {
-        "render": render_params,
-        "montage_stack": montage_stack,
-        "output_stack": output_stack,
-        "image_directory": tmp_dir,
-        "imgformat": "png",
-        "scale":0.1,
-        "zstart": 1020,
-        "zend": 1020
-    }
-    outjson = 'test_montage_scape_output.json'
-    #mod = MakeMontageScapeSectionStack(input_data=params, args=['--output_json', outjson])
-    #mod.run()
-
-    tagstr = "%s.0_%s.0" % (params['zstart'], params['zend'])
-    Z = [1020, 1020]
-
-    tilespecdir = os.path.join(params['image_directory'],
-                                   render_params['project'],
-                                   params['montage_stack'],
-                                   'sections_at_%s'%str(params['scale']),
-                                   'tilespecs_%s'%tagstr)
-
-    if not os.path.exists(tilespecdir):
-        os.makedirs(tilespecdir)
-
-    create_montage_scape_tile_specs(render,
-                                    params['montage_stack'],
-                                    params['image_directory'],
-                                    params['scale'],
-                                    render_params['project'],
-                                    tagstr,
-                                    params['imgformat'],
-                                    Z,
-                                    pool_size=pool_size)
-
-
-def test_make_montage_stack_module_without_downsamples(
-        render, montage_stack, tmpdir_factory):
-    # testing for make montage scape stack without having downsamples generated
-    tmp_dir = str(tmpdir_factory.mktemp('downsample'))
-    output_stack = '{}_Downsample'.format(montage_stack)
-    params = {
-        "render": render_params,
-        "montage_stack": montage_stack,
-        "output_stack": output_stack,
-        "image_directory": tmp_dir,
-        "imgformat": "png",
-        "scale": 0.1,
-        "zstart": 1020,
-        "zend": 1020
-    }
-
-    outjson = 'test_montage_scape_output.json'
-    mod = MakeMontageScapeSectionStack(
-        input_data=params, args=['--output_json', outjson])
-    mod.run()
-
-    tspecs = render.run(
-        renderapi.tilespec.get_tile_specs_from_stack, output_stack)
-
-    tsfn = urllib.parse.unquote(urllib.parse.urlparse(
-        tspecs[0].ip[0].imageUrl).path)
-    assert os.path.isfile(tsfn)
-    assert os.path.basename(tsfn) == '1020.0.png'
-
-def test_apply_rough_alignment_transform_with_scale(render, montage_stack, test_do_rough_alignment_with_scale, tmpdir_factory, prealigned_stack=None, output_stack=None):
```

Comment:
Why is this test removed?  If we don't want backwards-compatible scaling we should remove it from the module in another PR.

Reply:
I removed it without making it backwards-compatible... The python solver removes the scale regardless of introducing it in the downsample stack (@djkapner please correct me if I am wrong). 
Will re-introduce the matlab solving options then.

Reply:
the python solver doesn't actively do anything with scale.
I think what you're talking about is that the matlab solver did something like scale the transforms for downsampled images up, which is nice if you wanted to ndviz downsampled sections on top of high res sections.
I don't do that in the python one. If we wanted that, I don't think that is a solver operation.

Reply:
The scale factor was added for the AT team's operations as they wanted to see the ndviz downsampled section on top of montages. We have never used such a functionality. The one scenario where this test would fail is when we apply scale in downsampled stack and then the python solver ignores it and we end up applying transformations with scale in apply rough... 
The module needs  a change when we completely switch to python solver (again to be done in another PR)